### ISSUE TITLE
AttributeError: 'PytestPluginManager' object has no attribute 'addhooks'

### ISSUE THREAD

**Issue by ohing504** [pytest-dev/pytest-factoryboy#77]
'addhooks' attribute of PytestPluginManager is removed at [pytest 5.0.0](https://github.com/pytest-dev/pytest/commit/7513d87b153b67f6c1c6a8f491876bc0c7eb2e62).

Because of this, `add_hookspecs = getattr(pluginmanager, 'add_hookspecs',  pluginmanager.addhooks)` does not work (#16). It occur following error.

```text
Traceback (most recent call last):
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/bin/pytest", line 10, in <module>
    sys.exit(main())
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/_pytest/config/__init__.py", line 55, in main
    config = _prepareconfig(args, plugins)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/_pytest/config/__init__.py", line 200, in _prepareconfig
    pluginmanager=pluginmanager, args=args
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/hooks.py", line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/manager.py", line 87, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/manager.py", line 81, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/callers.py", line 203, in _multicall
    gen.send(outcome)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/_pytest/helpconfig.py", line 89, in pytest_cmdline_parse
    config = outcome.get_result()
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/_pytest/config/__init__.py", line 661, in pytest_cmdline_parse
    self.parse(args)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/_pytest/config/__init__.py", line 869, in parse
    self._preparse(args, addopts=addopts)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/_pytest/config/__init__.py", line 815, in _preparse
    self.pluginmanager.load_setuptools_entrypoints("pytest11")
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/manager.py", line 293, in load_setuptools_entrypoints
    self.register(plugin, name=ep.name)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/_pytest/config/__init__.py", line 302, in register
    ret = super().register(plugin, name)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/manager.py", line 121, in register
    hook._maybe_apply_history(hookimpl)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/hooks.py", line 336, in _maybe_apply_history
    res = self._hookexec(self, [method], kwargs)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/manager.py", line 87, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/manager.py", line 81, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/callers.py", line 208, in _multicall
    return outcome.get_result()
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/Users/ohing/.virtualenvs/django-drf-project-template-Kk8KycnX/lib/python3.7/site-packages/pytest_factoryboy/plugin.py", line 117, in pytest_addhooks
    add_hookspecs = getattr(pluginmanager, 'add_hookspecs',  pluginmanager.addhooks)
AttributeError: 'PytestPluginManager' object has no attribute 'addhooks'
```

**Comment by sidneijp** [comment]
Having the same trouble. Looks like something recent.

**Comment by raratiru** [comment]
Indeed all tests have failed due to this error:

https://travis-ci.com/raratiru/django-letsagree/builds/117579012

**Comment by MlopesD** [comment]
specify pytest==4.6.4 (previous version) in requirements.txt

**Comment by revmischa** [comment]
pls merge/release soon!

**Comment by youtux** [comment]
Fixed in #80. New release 2.0.3 is already available on pypi.
